package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-web3studio",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Should we up the version on every commit?

Reply:
It's a bit of a instinct thing TBH. 

What I would call 'best practice' is making single commits to do nothing but 'cut releases'. This is because, for projects being developed by many at once, it's hard to predict what you should change it to. 

Sense I know that I'm it, and I have merge/releasing abilities, I skipped a step. 

I've seen a friend of mine automate it with an emoji code in commit messages, but that was a lot. I've also released every pr by sticking a timestamp for the patch, but that broke semver 🤷‍♂️. Haven't found a solution to this that I really like.